### DISCUSSION TITLE
Add db/create db/drop console commands

### DISCUSSION THREAD

**Discussion by leevigraham** [craftcms/cms#15871] [Ideas]
It would be handy to have `db/create` and `db/drop` console commands added to Craft CMS.

Motivation:

We (Newism) create feature previews based on branches in our testing / CI environment. This involves creating new databases during the setup process and deleting the site / database when the branch is merged / deleted.

I'm currently doing this with my own hand rolled console command in a custom module. I like the approach of using the craft console because they .env variables are already loaded.

This could further be improved using the DB driver classes and the loaded DB config.

Symfony has these commands… I'm sure Laravel would have them as well.

![image](https://github.com/user-attachments/assets/ea1d1a9a-f9b0-41eb-9e30-828ff0d66465)

![image](https://github.com/user-attachments/assets/849f4b5c-ed5e-4637-87d7-6e13cd9dd525)

**Comment by Anubarak** [comment]
You already have those.
They are called
db/drop-all-tables https://craftcms.com/docs/5.x/reference/cli.html#db-drop-all-tables

And install

**Reply by brandonkelly** [reply]
Is that good enough @leevigraham? Or are you hoping for a command that drops the entire database?

**Reply by leevigraham** [reply]
That could work in place of `db/drop` … does it drop non-craft tables as well? A `db/create` would still be handy.
